api/office/schema.py

```diff
@@ -18,6 +22,12 @@ class Arguments:
 
     @Auth.user_roles('Admin')
     def mutate(self, info, **kwargs):
+        query = Location.get_query(info)
+        location = query.filter(
+            LocationModel.id == kwargs['location_id']).first()
+        admin_details = get_user_from_db()
+        if location.name != admin_details.location:
+            raise GraphQLError("You are not authorized to make changes in " + location.name)  # noqa: E501
```

Comment:
@peterwade153, I think we can do all of these in the helper and avoid code repetition. This will save us Code Climate issues.

Reply:
Thanks for that, currently abstracting after having the functionality working. That is being addressed

Reply:
Great! 👍